TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -3876,6 +3876,18 @@ Access control is discussed in detail in <doc:AccessControl>.
   Declarations marked with the `public` access-level modifier can also be accessed (but not subclassed)
   by code in a module that imports the module that contains that declaration.
 
+- term `package`:
+  Apply this modifier to a declaration
+  to indicate that the declaration can be accessed
+  only by code in the same package as the declaration.
+  A package is a unit of code distribution
+  that you define in the build system you're using.
+  When the build system compiles code,
+  it specifies the package name
+  by passing the `-package-name` flag to the Swift compiler.
+  Two modules are part of the same package
+  if the build system specifies the same package name when building them both.
```

Comment:
```suggestion
  if the build system specifies the same package name when building both of them.
```

Reply:
Changed to omit "both".  While applying the edit, I wasn't quite happy with either the new or old version here, and I think dropping "both" makes it less ambiguous.  The build system doesn't need to build both things at the same time — it just needs to specify the same package name when it builds each of them.